#!/usr/bin/env python3
# -*- coding: utf-8 -*-
# (c) ACE 

import os

class Config(object):
    # get a token from @BotFather
    pass
    """
    BOT_TOKEN = os.environ.get("BOT_TOKEN", "6655736303:AAHhiNLonJsr7wiCq_SWjF4St_AhdxqetyI")
    API_ID = int(os.environ["API_ID", 21659643]
    API_HASH = os.environ["API_HASH", "c5f065905ef47e693b47e0f60bb67217"]
    AUTH_USERS = "5487643307"""
# jisko bhi is repo ko deploy karne me error aa raha ho ( https://t.me/Stubborn1223 ) ye mere TG ka id ha message karke iska screen shot send kar dena me error solve kar dunga. 
